ngrx-immer is failing to build after release of latest version

Hi Tim,

ngrx-immer is failing to build with the following error :
73.01 Error: node_modules/ngrx-immer/store/index.d.ts:1:22 - error TS6053: File '/app/node_modules/ngrx-immer/store/index.ngtypecheck.d.ts' not found.
73.01 
73.01 1 /// <reference path="index.ngtypecheck.d.ts" />

The version currently used in package.json is "^2.0.1".
The build is failing after the new major version 2.2.2.
Can you please let me know the dependencies.

--- a/package.json
+++ b/package.json
@@ -25,7 +25,7 @@
 		"ts-node": "^10.9.1",
 		"tsconfig-paths": "^4.0.0",
 		"tsm": "^2.3.0",
-		"typescript": "5.4.5",
+		"typescript": "5.5.2",
 		"uvu": "^0.5.6"
 	}
 }

